README_CLOUDPAK.md

```diff
@@ -0,0 +1,137 @@
+# Db2 Event Store on Cloud Pak for Data
+
+In this readme, we will walk you through step by step instructions on how to use Db2 Event store on cloud pak for data.
+
+## Installing and provisioning service instances
+
+The services are not available by default. An administrator must install them on the IBM Cloud Pak for Data platform, and you must be given access to the service. To determine whether the service is installed, Click the Services icon (see below) and check whether the service that you require is enabled.
+
+![Service Icon](doc/source/images/service-icon-cpd.png)
+
+Following are the services that you need to get installed and provisioned.
+
+* Watson Studio
+* Db2 Event Store
+* Watson Machine Learning
+
+By default Jupyter notebooks are availabe part of Watson Studio. You need to make sure `Default Spark Pythyon 3.6 kernel` is available when creating notebooks. If not your administrator should help you install it.
```

Comment:
typos:  s/availabe/available/   s:/Pythyon/Python/
Add comma:  s/If not your administrator/If not, your administrator/